ffac-eol-ssid/luasrc/lib/gluon/upgrade/950-eol-ssid-nagger

```diff
@@ -11,18 +11,13 @@ if uci:get('eol-wifi', 'ssid') ~= nil then
     os.remove('/etc/config/eol-wifi')
 end
 
--- Get/initialize enabled state
-if site.eol_wifi_ssid() == nil then
-    -- Not applicable
-    os.exit(0)
+if not site.eol_ssid.enabled(false) -- disabled for site/domain
+or not uci:get_bool('eol-ssid', 'settings', 'enabled') -- disabled on router
```

Comment:
Isn't it uncommon that the site enabled and the uci enabled has to be set?
Or are we doing it this way in other packages too?

Reply:
We have specific domains that should not send the eol SSID, so the site configuration should overwrite what uci has set. 

And the other way round, if a site should send the eol SSID, a user should be able to turn it off inside their routers. 

That's what this piece of code should do.